dali/pipeline/operators/decoder/nvjpeg_decoder_decoupled_api.h

```diff
@@ -339,6 +339,7 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
   }
 
   USE_OPERATOR_MEMBERS();
+
```

Comment:
?

Reply:
Yeah, I automatically added `using Operator<Backend>::RunImpl` below every `USE_OPERATOR_MEMBERS();` but it shouldn't appear in mixed ops as they don't have the RunImpl at all. And I didn't see the error, as I had old nvjpeg when compiling so this file was ignored. Will clean up today.